### PR TITLE
fix(copy): Update the launcher copy

### PR DESCRIPTION
**Pull request by mindreeper2420**
Update the launcher copy to match the content given by UXD in https://docs.google.com/document/d/1PRKPlg9zOgG9Eu5GZr96GCADPhPnLwZte0jkVx5q3KQ/edit?usp=sharing

This also updates the demo overlays, which will need to be carried over to fabric8-ui.

### CODE REVIEW

**Comment by invincibleJai** [comment]
@mindreeper2420 I think we need to rebase it , it's showing lot of extra commits as well

**Comment by mindreeper2420** [comment]
@invincibleJai should be all good now - not sure why it grabbed all of those 🤷‍♂️

**Comment by invincibleJai** [comment]
[test]

**Comment by invincibleJai** [comment]
[test]

**Comment by invincibleJai** [comment]
@edewit can you pls take a look at this as it touching launcher flow.

Thanks @mindreeper2420, looks like u need to rebase. Sorry about that

**Comment by edewit** [comment]
Looks good to me

**Comment by invincibleJai** [comment]
@mindreeper2420 looks like tag got misplaced while rebase, I tried running it locally got below error

screen shot 2018-05-02 at 9 51 13 pm

**Comment by mindreeper2420** [comment]
@invincibleJai found it - the alignment of the `<div>` elements were off, so there were 2 closing tags that were missing.

**Comment by invincibleJai** [comment]
@mindreeper2420 nice even i was trying to find :)

File: src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.html [hunk 1]
```diff
@@ -114,10 +114,10 @@ <h2 class="card-pf-title">
           <div class="card-pf card-pf--small">
             <div class="card-pf-heading">
               <h2 class="card-pf-title">
-                Choose a runtime
+                Select a runtime
```
Comment (invincibleJai) [inline src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.html]: labels texts are not as per UXD which Brian has shared with us https://redhat.invisionapp.com/share/VGF1RF15J#/screens/286893069

Is it updated again?

File: src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.html [hunk 2]
```diff
@@ -6,13 +6,13 @@
           [debounce]="500">
         Select Mission & Runtime
       </h1>
-      <div class="col-xs-12 col-md-6 f8launcher-section-mission-runtime-desp">
+      <div class="col-xs-12 col-md-8 f8launcher-section-mission-runtime-desp">
```
Comment (invincibleJai) [inline src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.html]: Description ends over missions card only https://redhat.invisionapp.com/share/VGF1RF15J#/screens/286893069

File: src/app/launcher/import-app/gitprovider-importapp-step/gitprovider-importapp-step.component.html [hunk 3]
```diff
@@ -4,9 +4,6 @@
     <div class="container-fluid">
       <h1 snInViewport (onInViewportChange)="launcherComponent.onInViewportChange($event, 'GitProvider')"
           [debounce]="500">Authorize Git Provider</h1>
-      <p>
```
Comment (invincibleJai) [inline src/app/launcher/import-app/gitprovider-importapp-step/gitprovider-importapp-step.component.html]: it's there as per UXD https://redhat.invisionapp.com/share/VGF1RF15J#/screens/286893664

Comment (mindreeper2420) [reply src/app/launcher/import-app/gitprovider-importapp-step/gitprovider-importapp-step.component.html]: Per the copy docs (https://docs.google.com/document/d/1PRKPlg9zOgG9Eu5GZr96GCADPhPnLwZte0jkVx5q3KQ/edit?usp=sharing), that description should not be present. The copy docs are the final representation of what the text of the mockups should be. 120 screens is too much to keep updated text on, hence the copy docs :)

@catrobson @bdellasc

**Comment by mindreeper2420** [comment]
@invincibleJai the copy doc (https://docs.google.com/document/d/1PRKPlg9zOgG9Eu5GZr96GCADPhPnLwZte0jkVx5q3KQ/edit?usp=sharing) is the definitive document for any and all text.

As for the page layout issues, those are not from anything that I did as part of this PR.

**Comment by invincibleJai** [comment]
@mindreeper2420 Thanks Adam, needed your help here. I just tried master standalone the layouts were fine. Can you pls check once if you get time.

**Comment by mindreeper2420** [comment]
@invincibleJai should be all set now - I tested locally with the latest updates and the layout issue is no longer present.

**Comment by invincibleJai** [comment]
@mindreeper2420 thanks a lot :) let me test once.

**Comment by invincibleJai** [comment]
[test]

**Pull request merged**